library/registry

```diff
@@ -2,15 +2,12 @@ Maintainers: Milos Gajdos (@milosgajdos),
              Sebastiaan van Stijn (@thajeztah)
 GitRepo: https://github.com/docker/distribution-library-image.git
 
+Tags: 2.8.0, 2.8, 2, latest
+Architectures: amd64, arm64v8, arm32v6, arm32v7, ppc64le, s390x
+GitFetch: refs/heads/master
+GitCommit: 5fd2614beccba35242694b13f8a37653c05ea852
+
 Tags: 2.8.0-beta.1
```

Comment:
Any particular reason to keep this version listed here?  Removing tags here will remove them from the "Supported" section on the Hub readme (and will prevent us from spending cycles rebuilding them on the official build servers), but the tags will still be available to users who want them.  (See https://github.com/docker-library/official-images#library-definition-files for more detail on this.)

Reply:
No particular reason, we just followed the existing setup -- that being said, `beta` release is _identical_ from th stable one. I'm happy to remove it. Do you want me to?

Reply:
Removed https://github.com/docker-library/official-images/pull/11821/commits/dfb8057d207f261bf6011fb5fe98988b80799986